wolfcrypt/src/evp.c

```diff
@@ -6803,6 +6805,928 @@ void wolfSSL_EVP_PKEY_free(WOLFSSL_EVP_PKEY* key)
         }
     }
 }
+#if defined(OPENSSL_EXTRA)
+
+/* Indent writes white spaces of the number specified by "indents"
+ * to the BIO. The number of white spaces is limited from 0 to
+ * EVP_PKEY_PRINT_INDENT_MAX.
+ * returns the amount written to BIO.
+ */
+static int Indent(WOLFSSL_BIO* out, int indents )
+{
+    int i;
+    char space = ' ';
+
+    if (indents < 0) {
+        indents = 0;
+    }
+    else if (indents > EVP_PKEY_PRINT_INDENT_MAX) {
+        indents = EVP_PKEY_PRINT_INDENT_MAX;
+    }
+    if (out == NULL) {
+        return 0;
+    }
+    for (i = indents; i; i--) {
+        if (wolfSSL_BIO_write(out, &space, 1) < 0) {
+            break;
+        }
+    }
+    return indents -i;
+}
+/* DumpElement dump byte-data specified by "input" to the "out".
+ * Each line has leading white spaces( "indent" gives the number ) plus
+ * four spaces, then hex coded 15 byte data with separator ":" follow.   
+ * Each line looks like: 
+ * "    00:e6:ab: --- 9f:ef:"
+ * Parmeters:
+ * out     bio to output dump data
+ * input   buffer holding data to dump
+ * inlen   input data size
+ * indent  the number of spaces for indent
+ * Returns 1 on success, 0 on failure.
+ */
+static int DumpElement(WOLFSSL_BIO* out, const byte* input,
```

Comment:
Better name would be PrintHexWithColon

Reply:
Changed function name to PrintHexWithColon.